internal/pkg/cli/selector/selector.go

```diff
@@ -76,6 +87,66 @@ func NewWorkspaceSelect(prompt Prompter, store *config.Store, ws *workspace.Work
 	}
 }
 
+// NewDeploySelect returns a new selector that chooses services and environments from the deploy store.
+func NewDeploySelect(prompt Prompter, configStore *config.Store, deployStore *deploy.Store) *DeploySelect {
```

Comment:
this is my bad, but can we take as input interfaces to the constructors instead of structs?